bindings/java/src/main/native/jucx_common_def.cc

```diff
@@ -177,7 +177,10 @@ static inline void set_jucx_request_completed(JNIEnv *env, jobject jucx_request,
 {
     env->SetObjectField(jucx_request, native_id_field, NULL);
     if (ctx != NULL) {
-        env->SetLongField(jucx_request, sender_tag_field, ctx->sender_tag);
+        if (ctx->sender_tag != 0) {
```

Comment:
maybe pass constant from recv_callback? e.g "bool set_recv_info"
also, looking at recv_callback code, not clear who sets ctx->sender_tag?

Reply:
https://github.com/openucx/ucx/pull/5655/files#diff-aef6fcc6462b764657fa404971dd21e0R256 - it sets in tag_recv callback from tag_info

Reply:
The issue is that unlike length, tag could be any long value.  I've mentioned in docs:
`Sender tag, valid only for tag receive requests.`

Reply:
`/* sender_tag and length are initialized to 0, so try to avoid the overhead of setting them again */`